build/configs/rtl8730e/loadable_ext/defconfig

```diff
@@ -537,7 +539,35 @@ CONFIG_AUDIO_ALC1019=y
 #
 # LCD Driver Support
 #
-# CONFIG_LCD is not set
+CONFIG_LCD=y
+# CONFIG_LCD_HWCURSOR is not set
+# CONFIG_LCD_PACKEDMSFIRST is not set
+# CONFIG_LCD_UPDATE is not set
+
+#
+# Common Graphic LCD Settings
+#
+# CONFIG_LCD_FRAMEBUFFER is not set
+
+#
+# LCD driver selection
+#
+# CONFIG_LCD_NOGETRUN is not set
+# CONFIG_LCD_DMA_SUPPORT is not set
+
+#
+# Graphic LCD Devices
+#
+CONFIG_LCD_LANDSCAPE=y
+# CONFIG_LCD_PORTRAIT is not set
+# CONFIG_LCD_RPORTRAIT is not set
+# CONFIG_LCD_RLANDSCAPE is not set
+# CONFIG_LCD_ST7701 is not set
+CONFIG_LCD_XRES=240
+CONFIG_LCD_YRES=320
```

Comment:
Even you define this, you don't use them anywhere. If we will use lcd specific header file, then we can use the header instead of configs. BUT, if noone uses partial size of the lcd, then we can get hard-coded values in the lcd specific header.